examples/complete/example.yaml

```diff
@@ -0,0 +1,22 @@
+datadog_api_key: ENC[AES256_GCM,data:joPOPI58VO5E2g==,iv:G3BamrS3ANDMhJqjyZbn1YhAqf2GJ7g7DdVivrYVzDw=,tag:k2sGvVtoxznn7U8x8I7oUw==,type:int]
+datadog_app_key: ENC[AES256_GCM,data:GeerHEomnls2Cg==,iv:9uiNp5vvv/8/6sqNix28FRm1btWy6CF3fnGD3RV3oMI=,tag:LbGRJu7Bz8HdJYR+6iO/vQ==,type:int]
```

Comment:
Did you create a test datadog account? Because if so... bravo 😁

Reply:
I'm all for using `age` here, makes sense 👍  

We should spin up a internal DD org so we can test this module, even if those are tests are manual for now. We could also create some `terraform test` blocks as well... 

Let's talk about this Thursday.